BUILD.gn

```diff
@@ -59,6 +59,12 @@ executable("aquarium") {
     "third_party/imgui",
     "third_party/imgui/examples",
     "third_party/glfw/include",
+    "third_party/cxxopts/include",
+  ]
+
+  defines = [
+    "CXXOPTS_NO_EXCEPTIONS",
```

Comment:
Do you mean unrecognized option or value of option in bad format? I think we can do more on these two situations.

Reply:
For example:

```
aquarium.exe --backend d3d12 --msaa-sample-count foobar
Argument 'foobar' failed to parse
```

In such a case, `options.parse()` will not return, so we would lose the opportunity to print `options.help()` apart from the error message.

Reply:
This is because I restricted it to be integer as below:
("msaa-sample-count", "specifies sample count of MSAA, while 1 for non-MSAA", cxxopts::value<int>()).

If you hope to parse it by yourself, you may remove "cxxopts::value<int>()" here, and write your own parser.

Reply:
I'm not asking for a custom parser. I just feel the behavior of throw_or_mimic() is horrible when exceptions are disabled.